fabtests/common/shared.c

```diff
@@ -538,6 +538,10 @@ int ft_alloc_msgs(void)
 			   MAX(rx_size, FT_MAX_CTRL_MSG) * opts.window_size;
 	}
 
+	/* Allow enough space for RMA to operate in a distinct memory
+	   region that ft_sync() won't touch */
+	buf_size += 4 + MAX( ft_tx_prefix_size(), ft_rx_prefix_size() );
```

Comment:
4 is a magic number.  Can we use sizeof(whatever 4 is supposed to represent)?

Reply:
I introduced #define FT_RMA_SYNC_MSG_BYTES in shared.h.

There is some funny business.  ft_sync() itself uses only 1 byte, and again I'm hesitant to change that, but ft_rma_comp() routines use 4 bytes, I think arbitrarily.  I've replaced the RMA completion functions that do 4-byte sends so they now use the new constant.

This patch uses both ft_sync and rma_comp, so the requirement is that this constant be >= the maximum of all those message sizes.